src/reversion/models.py

```diff
@@ -214,3 +194,6 @@ def __str__(self):
     #Meta
     class Meta:
         app_label = 'reversion'
+        index_together = [
+            ["content_type", "object_id"],
```

Comment:
Because content_type is a ForeignKey, it'll have an index anyway, so I think all bases are covered.